rest-service/manager_rest/workflow_client.py

```diff
@@ -15,11 +15,26 @@
 
 
 from flask import current_app
-
+from flask_securest.rest_security import SECURED_MODE
 from manager_rest import celery_client
+from manager_rest import config
 
 
 class WorkflowClient(object):
+    @staticmethod
+    def _get_rest_credentials():
+        if not current_app.config.get(SECURED_MODE):
+            return {}
+
+        if hasattr(current_app, 'auth_token_generator'):
+            return {'rest_token':
```

Comment:
we decided to increase the default expiration to 24h as this should cover all uses